pages/common/git-fork.md

```diff
@@ -0,0 +1,13 @@
+# git fork
+
+> Fork the given GitHub repo. Like `git clone` but forks first.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-fork>.
+
+- Fork the specified GitHub repo by full URL:
```

Comment:
```suggestion
- Fork and clone a specific GitHub repo by full URL:
```
Now it uses the same wording as the second example.

Reply:
Just dropping my thoughts:
* The word specific/specified redundant if it'll that it's specified by a unique property anyway. ("by URL").
* The word "full" is redundant, a URL is assumed to be ~full~ _absolute_ unless specified otherwise.

```suggestion
- Fork and clone a GitHub repository by its URL:
```

Thoughts on that, or disagree? @marchersimon

Reply:
Actually that makes sense. And it also sounds a lot nicer.

Reply:
Yep, that's better. 2nd suggestion it is